client/src/org/adempiere/Adempiere.java

```diff
@@ -187,7 +195,8 @@ private static void setPackageInfo()
 		if (s_ImplementationVendor != null)
 			return;
 
-		Package adempierePackage = Package.getPackage("org.adempiere");
+		Class<?> clazz = Adempiere.class;
```

Comment:
The editor tells me that it is in a depreciated form, the form that I had placed was safer because when making a class change would indicate that the path of the import was changed, and not as a static string, however you are not right because it has nothing to do with the solution.